docs/050-breaking-changes.rst

```diff
@@ -127,7 +127,11 @@ For most of the topics the compiler will provide suggestions.
   adjusted within the type before the conversion.  For example, you can convert
   a ``bytes4`` (4 bytes) to a ``uint64`` (8 bytes) by first converting the
   ``bytes4`` variable to ``bytes8`` and then to ``uint64``. You get the
-  opposite padding when converting through ``uint32``.
+  opposite padding when converting through ``uint32``. Before v0.5.0 ``uintY(bytesX(foo))``
```

Comment:
This is a very useful comment, but I think it can also be a bit confusing. Can you maybe summarize the two conversions first by saying something like "any conversion between `bytesX` and `uintY` would go through `uint(8X)` first.". Also writing `uintX(bytesX(foo))` could be taken as two conversions, depending on the type of `foo`.

Reply:
Good idea. I have changed the comment. Now is shorter and concise.